models/receipt.go

```diff
@@ -31,6 +31,7 @@ type StorageReceipt struct {
 	BlockNumber       *big.Int         `json:"blockNumber,omitempty"`
 	TransactionIndex  uint             `json:"transactionIndex"`
 	RevertReason      []byte           `json:"revertReason"`
+	PrecompiledCalls  []byte
```

Comment:
:warning: Reminder to re-index, upon deploying a release that contains this change :warning:
You will also need to set this value, upon decoding the `EVM.TransactionExecuted` event, here: https://github.com/onflow/flow-evm-gateway/blob/main/models/transaction.go#L200

Reply:
Updated